### PR TITLE
remove profile

### PR DESCRIPTION
**Pull request by tomerd**
motivation: reduce complexity

changes: remove profile model and update API and storage accordingly

### CODE REVIEW

**Comment by tomerd** [comment]
@swift-ci please smoke test

File: Tests/PackageCollectionsTests/PackageCollectionsTests.swift [hunk 1]
```diff
@@ -90,21 +116,14 @@ final class PackageCollectionsTests: XCTestCase {
...
         }
-
-        do {
-            let unknownProfile = PackageCollectionsModel.Profile(name: "profile-\(UUID().uuidString)")
-            XCTAssertThrowsError(try tsc_await { callback in packageCollections.removeCollection(mockCollections[mockCollections.count - 2].source, from: unknownProfile, callback: callback) }, "expected error")
-        }
     }
 
-    func testDeleteFromStorageWhenLast() throws {
+    func testDeleteFromFromBothStorages() throws {
```
Comment (yim-lee) [inline Tests/PackageCollectionsTests/PackageCollectionsTests.swift:121]:
```suggestion
    func testDeleteFromBothStorages() throws {
```

**Review by yim-lee — APPROVED** [review]
👍

File: Tests/PackageCollectionsTests/PackageCollectionsTests.swift (hunk 1 above)
Comment (tomerd) [reply Tests/PackageCollectionsTests/PackageCollectionsTests.swift:121]: will address in #3069 so we dont have to run CI again for one typo

**Pull request merged**